### ISSUE TITLE
Error (lazydata failed) when installing on Debian 11

### ISSUE THREAD

**Issue by justinchuntingho** [chainsawriot/oolong#67]
I came across an error during installation on a Debian 11 machine. I tried both CRAN and Github but the same error persists.

```
> install.packages("oolong")
Installing package into ‘/home/user/R/x86_64-pc-linux-gnu-library/4.0’
(as ‘lib’ is unspecified)
trying URL 'https://cloud.r-project.org/src/contrib/oolong_0.4.1.tar.gz'
Content type 'application/x-gzip' length 3535091 bytes (3.4 MB)
==================================================
downloaded 3.4 MB

* installing *source* package ‘oolong’ ...
** package ‘oolong’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** data
*** moving datasets to lazyload DB
Warning: namespace ‘text2vec’ is not available and has been replaced
by .GlobalEnv when processing object ‘abstracts_warplda’
Warning: namespace ‘mlapi’ is not available and has been replaced
by .GlobalEnv when processing object ‘abstracts_warplda’
Warning: namespace ‘text2vec’ is not available and has been replaced
by .GlobalEnv when processing object ‘abstracts_warplda’
Warning: namespace ‘mlapi’ is not available and has been replaced
by .GlobalEnv when processing object ‘abstracts_warplda’
Error in normalize(self$components, "l1") : 
  could not find function "normalize"
ERROR: lazydata failed for package ‘oolong’
* removing ‘/home/user/R/x86_64-pc-linux-gnu-library/4.0/oolong’
Warning in install.packages :
  installation of package ‘oolong’ had non-zero exit status

The downloaded source packages are in
	‘/tmp/RtmpXsORvV/downloaded_packages’
```

Here's my session info:
```
R version 4.0.4 (2021-02-15)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Debian GNU/Linux 11 (bullseye)

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/atlas/libblas.so.3.10.3
LAPACK: /usr/lib/x86_64-linux-gnu/atlas/liblapack.so.3.10.3

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8   
 [6] LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_4.0.4 tools_4.0.4   
```

**Comment by justinchuntingho** [comment]
Update: I just tried on RStudio Cloud, same error:

session info:
```
R version 4.1.3 (2022-03-10)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.4 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/atlas/libblas.so.3.10.3
LAPACK: /usr/lib/x86_64-linux-gnu/atlas/liblapack.so.3.10.3

locale:
 [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8        LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   
 [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C           LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_4.1.3 tools_4.1.3  
```

**Comment by chainsawriot** [comment]
@justinchuntingho 

tl;dr - short term solution

```r
install.packages("oolong", dependencies = TRUE)
```

But in the long run, I should kick the data ‘abstracts_warplda’
